scripts/spec.json

```diff
@@ -0,0 +1,25 @@
+{
+    "title": "Space Acres",
+    "background": "background.png",
+    "icon-size": 100,
+    "window": {
+        "size": {
+            "width": 500,
+            "height": 300
+        }
+    },
+    "contents": [
+        {
+            "x": 150,
+            "y": 130,
+            "type": "file",
+            "path": "../target/bundle/universal/SpaceAcres.app"
+        },
+        {
+            "x": 350,
+            "y": 130,
+            "type": "link",
+            "path": "/Applications"
+        }
+    ]
+}
```

Comment:
NIT: All files should end with a single new line

Reply:
Added new line at the end of file